account_bank_statement_import_mt940_base/mt940.py

```diff
@@ -83,14 +83,28 @@ def handle_common_subfields(transaction, subfields):
     for counterpart_field in ['CNTP', 'BENM', 'ORDP']:
         if counterpart_field in subfields:
             get_counterpart(transaction, subfields[counterpart_field])
+    if not transaction.message:
+        transaction.message = ''
     # REMI: Remitter information (text entered by other party on trans.):
     if 'REMI' in subfields:
-        transaction.message = (
-            '/'.join(x for x in subfields['REMI'] if x))
+        transaction.message += (
+            subfields['REMI'][2]
+            # this might look like
+            # /REMI/USTD//<remittance info>/
+            # or
+            # /REMI/STRD/CUR/<betalingskenmerk>/
+            if len(subfields['REMI']) >= 3 and subfields['REMI'][0] in [
+                'STRD', 'USTD'
+            ]
+            else
+            '/'.join(x for x in subfields['REMI'] if x)
+        )
+    # EREF: End-to-end reference
```

Comment:
I think EREF should not be put into message (which will get into name), because in many cases it will lead to duplicate information. Eref will already end up in the ref field of the bank statement line.